phpstan.neon

```diff
@@ -2,3 +2,5 @@ parameters:
     inferPrivatePropertyTypeFromConstructor: true
     excludes_analyse:
         - vendor
+    ignoreErrors:
+        - '#Access to an undefined property SportTrackingDataSdk\\ClientHydra\\Proxy\\ProxyObject::\$id\.#'
```

Comment:
cette erreur sort parce qu'on fait ça dans le proxy object :

`        // Id special case
        if ('id' === $property) {
            return $this->{$property};
        }`

sauf que par défaut les proxy object ont pas de propriété $id, en revanche toutes nos entités qui l'étendent l'ont bien, donc j'ai préféré ajouter l'exception dans phpstan